service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -300,15 +300,15 @@ void getDatasetPreviewTableRepo() {
                                     new ColumnModel()
                                         .datatype(TableDataType.INTEGER)
                                         .name("column a"))))));
-    when(datarepoService.getPreviewTable(user, tdrDataset.storageSourceId(), tableName))
+    when(datarepoService.getPreviewTable(user, tdrDataset.storageSourceId(), tableName, 30))
```

Comment:
Using `30` here makes this test dependent on the constant `MAX_ROWS` without referring to it directly. Instead this should either refer to the constant (by making the constant visible to this code by removing `private`) or by changing the mock argument list to use argument matchers. Using an argument matcher would look something like:
```suggestion
    when(datarepoService.getPreviewTable(eq(user), eq(tdrDataset.storageSourceId()), eq(tableName), anyInt()))
```